tests/tests.sh

```diff
@@ -46,10 +55,13 @@ test_with_default_tag() {
 }
 
 test_download_cwp_version() {
-  default_cwp_jar=$(download_cwp "$working_directory")
+  test_download_working_directory="$working_directory/test_download_cwp_version"
+  rm -rf "$test_download_working_directory"
+  mkdir "$test_download_working_directory"
+  default_cwp_jar=$(download_cwp "$test_download_working_directory")
```

Comment:
Maybe this piece of code is not part of the scope for this task, but this test was causing some flakiness and I think this change avoids it